src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,12 +6,13 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    public static final String ADDRESS_PREFIX = "Address: ";
```

Comment:
Is `public` the best access modifier for `ADDRESS_PREFIX` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.